editoast/src/views/timetable.rs

```diff
@@ -449,21 +458,49 @@ impl Conflict {
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
-struct PacedTrainOccurrenceId {
-    paced_train_id: i64,
-    index: u64,
+#[serde(untagged)]
+enum ConflictPacedTrainId {
```

Comment:
Why renaming `PacedTrainOccurrenceId` into `ConflictPacedTrainId`? It's still a struct that allows us to reference a specific occurrence among all the occurrences of a paced train if I understand correctly, not something specific about conflicts.